src/devices/Amg88xx/Amg88xxUtils.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
We've just moved to 2 lines headers. If you can adjust, that would be perfect.

Reply:
(remove third line, no change on the first two)

Reply:
Done for all .cs files.